microsoft/react-native-macos
# Implemented AccessibilityInfo.announceForAccessibility

#### Please select one of the following

## Summary

When the MacOS fork was made all the AccessibilityInfo functionality was stubbed out because the iOS Accessibility APIs are different from the MacOS APIs. I've filled back out the functionality for announceForAccessibility.

## Changelog

[MacOS] [Added] - Implemented AccessibilityInfo.announceForAccessibility

## Test Plan

1. Run RNTester
2. Go to the accessibility page
3. Scroll to Announce for Accessibility (the second to last entry)
4. Toggle VoiceOver in settings.
5. Move VoiceOver Focus to "Announce for Accessibility"
6. Hit [Space]
- You should see/hear Voice Over announce "Announcement Test"
<img width="639" alt="Screen Shot 2020-06-22 at 14 43 37" src="https://user-images.githubusercontent.com/65255457/85338311-e4816200-b496-11ea-8017-79d779dd9e4b.png">



###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-macos/pull/470)
